delivery_app/models.py

```diff
@@ -1,3 +1,136 @@
 from django.db import models
+from delivery_app.utils import calculate_distance 
 
-# Create your models here.
+
+class Location(models.Model):
+    address = models.CharField(max_length=64)
+    latitude = models.FloatField()
+    longitude = models.FloatField()
+
+    class Meta:
+        verbose_name = "Location"
+        verbose_name_plural = "Locations"
+
+    def __str__(self):
+        return self.address
+
+
+class Order(models.Model):
+    order_id = models.CharField(max_length=64, unique=True)
+    weight = models.FloatField(help_text="Weight in kilograms")
+    delivery_location = models.ForeignKey(
+        Location,
+        on_delete=models.CASCADE
+    )
+    date_of_order = models.DateField()
+
+    class Meta:
+        verbose_name = "Order"
+        verbose_name_plural = "Orders"
+
+    def __str__(self):
+        return f"Order {self.order_id} - {self.weight} kg"
+
+
+class Distance(models.Model):
```

Comment:
PLease tell the purpose of this model

Reply:
The Order model represents the order's details placed by a customer.